Sources/SwiftParser/Lexer/Cursor.swift

```diff
@@ -242,6 +242,9 @@ extension Lexer {
 
     /// If we have already lexed a token, the kind of the previously lexed token
     var previousTokenKind: RawTokenKind?
+    
+    /// If we have already lexed a token, stores whether the previous lexeme‘s ending contains a newline.
+    var previousLexemeTrailingNewlinePresence: NewlinePresence?
```

Comment:
Hello @ahoppen!
When I modified the parsing logic so that newlines in multiline strings also have the isAtStartOfLine flag, I used this property together.
So I changed the name to something that could include both trailing trivia and multiline string parsing, but would it be better to split them up and have them as separate properties? 🤔 
May I ask for your opinions?

Reply:
That’s a good name. I like it!